client/extensions/woocommerce/app/reviews/review-reply.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External depedencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { getReviewReply } from 'woocommerce/state/sites/review-replies/selectors';
+import Gravatar from './gravatar';
+import humanDate from 'lib/human-date';
+
+class ReviewReply extends Component {
+	static propTypes = {
+		siteId: PropTypes.number,
+		reviewId: PropTypes.number.isRequired,
+		replyId: PropTypes.number.isRequired,
+	};
+
+	renderReplyActions = () => {
+		const { translate } = this.props;
+		return (
+			<div className="reviews__reply-actions">
+				<Button borderless className="reviews__reply-action-edit">
+					<Gridicon icon="pencil" />
+					<span>{ translate( 'Edit reply' ) }</span>
+				</Button>
+				<Button borderless className="reviews__reply-action-delete">
+					<Gridicon icon="trash" />
+					<span>{ translate( 'Delete reply' ) }</span>
+				</Button>
+			</div>
+		);
+	}
+
+	render() {
+		const { reply } = this.props;
+		const content = reply.content && reply.content.rendered || '';
+		return (
+			<div className="reviews__reply">
+				<div className="reviews__reply-bar">
+					<div className="reviews__author-gravatar">
+						<Gravatar
+							object={ reply }
+							forType="reply"
+						/>
+					</div>
+
+					<div className="reviews__info">
+						<div className="reviews__author-name">{ reply.author_name }</div>
+						<div className="reviews__date">{ humanDate( reply.date_gmt + 'Z' ) }</div>
+					</div>
+
+					{ this.renderReplyActions() }
+				</div>
+
+				<div className="reviews__reply-content"
+					dangerouslySetInnerHTML={ { __html: content } } //eslint-disable-line react/no-danger
```

Comment:
Interesting - this does mean things like img can make it through, e.g.

```
Testing
<b>Bold</b>

<img src="https://upload.wikimedia.org/wikipedia/commons/c/c4/PM5544_with_non-PAL_signals.png" />

<h1>Boo</h1>

<p>Y'all</p>
```

yields

<img width="1303" alt="screen shot 2017-09-15 at 1 44 59 pm" src="https://user-images.githubusercontent.com/1595739/30502713-458c91e8-9a1c-11e7-887d-c254602c037f.png">

Reply:
But since the tags allowed are limited, I guess that is OK